File: translation_output/no_test/llama2/codenet/java/python-sanitized/p02723.py
import java.util

class p02723:
    def main(self, args):
        sc = java.util.Scanner(System.in)
        s = sc.next()
        if s[2] == s[3] and s[4] == s[5]:
            print("Yes")
        else:
            print("No")

File: translation_output/w_test/starcoder/codenet/python/java/p02723.java
<fim_prefix>Translate the following python code to java and enclose your solution inside ```java```.
A sample test case is provided below:

Test input:
sippuu

Expected output:
Yes


```
s = input()
if s[2] == s[3] and s[4] == s[5]:
    print("Yes")
else:
    print("No")
```

java code:<fim_suffix><fim_middle>
```java
import java.util.*;
public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String s = sc.next();
        if (s.charAt(2) == s.charAt(3) && s.charAt(4) == s.charAt(5)) {
            System.out.println("Yes");
        } else {
            System.out.println("No");
        }
    }
}
```
'''
<|endoftext|>
